insert_string.h

```diff
@@ -0,0 +1,81 @@
+#ifndef INSERT_STRING_H_
+#define INSERT_STRING_H_
+
+/* insert_string.h -- Private insert_string functions shared with more than
+ *                    one insert string implementation
+ *
+ * Copyright (C) 1995-2013 Jean-loup Gailly and Mark Adler
+ * For conditions of distribution and use, see copyright notice in zlib.h
```

Comment:
I am pretty sure that quick_insert_string was not made by the two above, does this copyright notice need additions?

Reply:
Fixed.